packages/components/src/Tooltip/tests/chromatic/Tooltip.stories.tsx

```diff
@@ -0,0 +1,242 @@
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Grid>
+    )
+} satisfies Story;
+
+export const ShouldFlip = {
+    render: args => (
+        <Stack>
+            <H1>Original Placement: left</H1>
+            <TooltipTrigger defaultOpen placement="left">
+                <Button>{buttonText}</Button>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        </Stack>
+    ),
+    decorators: [
+        Story => (
+            <Flex justifyContent="left">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const LinkTrigger = {
+    render: args => (
+        <TooltipTrigger defaultOpen>
+            <Link>{buttonText}</Link>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    )
+} satisfies Story;
+
+export const AvatarTrigger = {
+    render: function Render(args) {
+        return (
+            <TooltipTrigger defaultOpen>
+                <PassiveTrigger>
+                    <Avatar name="Fred Allen" />
+                </PassiveTrigger>
+                <Tooltip {...args} />
+            </TooltipTrigger>
+        );
+    }
+} satisfies Story;
+
+export const LongContent = {
+    render: args => (
+        <TooltipTrigger isOpen>
+            <Button>{buttonText}</Button>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    ), 
+    args: {
+        children: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam in turpis
+                        ac libero tincidunt hendrerit. Ut vitae nisl nec orci laoreet tristique.
+                        Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla
+                        facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi. Nulla facilisi.`
+    },
+    decorators: [
+        Story => (
+            <Flex UNSAFE_marginTop="8rem" justifyContent="center">
+                <Story />
+            </Flex>
+        )
+    ],
+    parameters: {
+        skipGlobalDecorator: true
+    }
+} satisfies Story;
+
+export const DisabledTrigger = {
+    render: args => (
+        <TooltipTrigger>
+            <PassiveTrigger data-testid="passive-trigger">
+                <Button isDisabled>{buttonText}</Button>
+            </PassiveTrigger>
+            <Tooltip {...args} />
+        </TooltipTrigger>
+    ),
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        const trigger = canvas.getAllByTestId("passive-trigger")[0];
+        const trigger2 = canvas.getAllByTestId("passive-trigger")[1];
+        // For some reason, we need to hover over the second trigger first
+        await userEvent.hover(trigger2);
+        await userEvent.hover(trigger);
+        await waitFor(async () => {
+            await expect(screen.getByText(childrenText)).toBeVisible();
+        });
+    }
+} satisfies Story;
+
+export const DisabledListItem = {
```

Comment:
This story should be in the listbox.stories.tsx. Tooltip is going to be used in other components